jobserver/management/commands/release.py

```diff
@@ -0,0 +1,26 @@
+import sys
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from jobserver.models import Backend, Workspace
+from jobserver.releases import create_upload_zip, handle_release
+
+
+class Command(BaseCommand):
+    help = "Configure admins to "
```

Comment:
Late seagulling but not sure this help is very useful :)

Could we describe when you might want to use it? 

It might also warrant a sentence or two in `DEVELOPERS.md`?

It's so common that I make a management command that's useful for development, everyone forgets about it, two years later someone removes it because "no-one uses it", very mild sadness ensues

Reply:
Hah! That's entirely my fault, I think the `ensure_admins` command has the same "help".

Reply:
Oh, sorry, didn't notice that c/p from the admin command. Will fix in next PR